### ISSUE TITLE
Move loose barrier=gate/lift_gate to z18

### ISSUE THREAD

**Issue by kocio-pl** [gravitystorm/openstreetmap-carto#3122]
Similar to #2756.

Currently we show all the gates from z16 and it may be important to see them so early (I'm not sure about it), but at least we should render loose gates (not included in any road) from z18. This would promote more complete tagging (adding roads), which would be good for navigation, and avoid rendering clutter at the same time.

See the [example](https://www.openstreetmap.org/node/945932335#map=16/46.3636/11.0309) (Italy - Cles, z16):
![screenshot-2018-3-15 dane obiektu openstreetmap](https://user-images.githubusercontent.com/5439713/37461443-a9cd71d4-284e-11e8-9dc7-c6032bdde73c.png)

**Comment by kocio-pl** [comment]
By the way - does anybody have a clue why do we need such barriers on z16+?

**Comment by SomeoneElseOSM** [comment]
By "such barriers" do you mean "any gates at all" on z16+ or "gates on roads, paths or tracks" at z16+?  I can see the logic of displaying gates in hedges etc. but not on paths later than gates actually on paths (though naturally being less urban-centric than most I'd say "17 instead of 15" rather than "18 instead of 16" - but that's a different discussion altogether).
How technically would you do "this gate is part of a highway so render it sooner" (apologies if this is offtopic; I'm asking because it'd be good to do that at e.g. https://map.atownsend.org.uk/maps/map/map.html#zoom=15&lat=53.16392&lon=-1.38033 ).

**Comment by kocio-pl** [comment]
I mean both types from the title, however there can be more of them - especially with the new icons designed lately: https://github.com/gravitystorm/openstreetmap-carto/issues/846#issuecomment-371893133.

Technically I've been given a hint to look at the turning circle code - it's a bigger SQL statement, so I don't see the immediate pattern, but maybe somebody can use it:
https://github.com/gravitystorm/openstreetmap-carto/blob/94e46239a658ffd2abbe866b4bdc553c69c9d5c8/project.mml#L639-L666

**Comment by kocio-pl** [comment]
I'm getting more and more convinced to start with moving all such barriers to z17+ (and later move detached ones to z18+). What do you think about it?

**Comment by kocio-pl** [comment]
See also #323 - quite a lot of discussion to read, but maybe we could start with moving all the gates to z17+.

**Comment by kocio-pl** [comment]
See also #1161 - could be done as a simple rework of both color and zoom level.

**Comment by lakedistrictOSM** [comment]
> I'm getting more and more convinced to start with moving all such barriers to z17+ (and later move detached ones to z18+). What do you think about it?

z16 is generally too early for detached and attached barriers in both urban and rural areas. z17+ would work better.

**Comment by kocio-pl** [comment]
I think also about moving bollards to z17+. I also expect them to be on local roads, so lower zoom is not needed for showing them.

**Comment by Adamant36** [comment]
@kocio-pl, are you still interested in moving gates to z18? If so, I'm totally willing to do a PR for it. Along with bollards.

**Comment by kocio-pl** [comment]
After #3157 (moving barriers from z16+ to z17+) it does not bug me, because there are more obvious thins I focus on. But if you think it's worth more attention, feel free to act.

**Comment by Adamant36** [comment]
I guess its OK for now. Want to close the issue since its essentially dealt with?

**Comment by kocio-pl** [comment]
OK, it makes sense.

**Comment by jeisenbe** [comment]
I would prefer to keep them on z17

**Comment by Adamant36** [comment]
Yeah, I think they are fine there. That's why I had Kocio-pl close the issue.

**Comment by jeisenbe** [comment]
Sorry, my comment took a long time to post (30min). Slow internet today.

**Comment by Adamant36** [comment]
@jeisenbe, oh. No worries. It sucks your internet is that slow.

**Comment by kocio-pl** [comment]
I'm impressed how much you do with such a bad connection...
